client-app/shared/catalog/components/product/variations.vue

```diff
@@ -52,6 +52,10 @@ interface IProps {
 }
 
 defineProps<IProps>();
+
+function filteredProperties(properties: Property[]) {
+  return properties.filter((prop) => !!prop.value).slice(0, 3);
```

Comment:
Good job! The only think we might want to consider - can our prop value be explicitly set to `0`?
If so, maybe it's better to use explicit check on `undefuned`, `null` and empty string, otherwise we never see `0` value in product variations.

Reply:
fixed with another bug